src/app/components/home/home.component.ts

```diff
@@ -27,7 +27,7 @@ import {XLarge} from './directives/x-large';
   // Every Angular template is first compiled by the browser before Angular runs it's compiler
   template: require('./home.html')
 })
-export class Home {
+export class HomeCmp implements OnInit {
```

Comment:
Even though using `ngOnInit` will work without implementing `OnInit`, it appears to be recommended to still implement for TypeScript sake:
https://github.com/angular/angular/issues/5814